elastic4s-core/src/main/scala/com/sksamuel/elastic4s/requests/indexes/MappingHandlers.scala

```diff
@@ -3,7 +3,7 @@ package com.sksamuel.elastic4s.requests.indexes
 import com.sksamuel.elastic4s._
 import com.sksamuel.elastic4s.requests.mappings.{GetFieldMappingRequest, GetMappingRequest, PutMappingRequest}
 
-case class IndexMappings(index: String, mappings: Map[String, Any])
+case class IndexMappings(index: String, mappings: Map[String, Any], meta: Map[String, Any] = Map.empty[String, Any])
```

Comment:
Is it ok to give the meta field in `IndexMappings` a default value? I thought this way it wouldn't break existing code?

Reply:
Yes makes sense.